src/generators/web/ui/index.css

```diff
@@ -70,3 +70,17 @@ main {
     }
   }
 }
+
+#modalContent li > div > a {
+  > svg {
+    flex-shrink: 0;
+  }
+  > div {
+    min-width: 0;
+    > p {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  }
+}
```

Comment:
Note: This can technically be done in `@node-core/ui-components`, would we prefer that?

It'll have no effect on the website, except for that it'll create unused CSS code.